nilearn/plotting/displays.py

```diff
@@ -870,8 +945,13 @@ def annotate(self, left_right=True, positions=True, size=12, **kwargs):
         positions: boolean, optional
             If positions is True, annotations indicating the
             positions of the cuts are drawn.
+        scalebar: boolean, optional
+            If scalebar is True, cuts are annotated with a reference
+            scale bar (bottom-right position by default).
```

Comment:
Here you should reference the method  "draw_scale_bar" for finer control of the scale bar.